guides/theming.md

```diff
@@ -367,6 +367,83 @@ $my-palette: mat.define-palette(mat.$indigo-palette);
 }
 ```
 
+## Strong focus indicators
+
+By default, most components indicate browser focus by changing their background color as prescribed
```

Comment:
Should it be "described" rather than "prescribed"?

Reply:
"Prescribed" is what I _intended_, for the definition "state authoritatively or as a rule that (an action or procedure) should be carried out." Is it too weird?

Reply:
As a non-native speaker, I've only heard "prescribed" as something somebody strongly recommends to you (e.g. prescribed medicine), but it might just be me.

Reply:
Changed to "described"